docs/personalization/recommendation_models.md

```diff
@@ -124,6 +129,18 @@ An element added to this list will never be recommended so it must be treated wi
 Pseudo recommendation model that shows the user products from their own history. 
 For example, the "You have just watched" box.
 
+#### Recurring purchase
+
+Recognizes purchase patterns and returns recommendations for items where these pattern can occur.
+This model is based on predictable purchase occuring at regular intervals going forward with a relatively high degree of certainty.
+Model recommends items which are suitable for this pattern. Item recommendations are displayed 20% before the estimated pattern of repeating purchasing the product.
```

Comment:
```suggestion
The model recommends items which are suitable for this pattern. Item recommendations are displayed 20% before the estimated pattern of repeating purchasing the product.
```
"are displayed 20% before" - that's somehow awkward, even with the explanation below